src/common/data-store/index.js

```diff
@@ -0,0 +1,95 @@
+import browser from "webextension-polyfill";
+
+import { encryptData, decryptData } from "../../common/lib/crypto";
+
+/**
+ * Encripted storage module. A passord is required to initialize this module.
+ * The storage is under the form of key-value pairs.
+ */
+let cachedStorage = null;
+
+function init(password) {
+  if (!password) {
+    throw new Error("A passord is required to initialize the storage module!");
```

Comment:
I see quite some exceptions of this in my console. Should this really be an error?

Reply:
agree, log&return